CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+<!-- SPDX-License-Identifier: CC0-1.0 -->
+# Contributing to Govdirectory
+
+🙇‍♀️ Thank you for contributing!
+
+We love issues and pull requests from everyone.
+If you're not comfortable with those features of GitHub, you can also just [start a discussion](https://github.com/govdirectory/website/discussions).
+
+## Problems, suggestions and questions in issues
+
+Please help development by reporting problems, suggesting changes and asking questions.
+To do this, you can [create a GitHub issue](https://help.github.com/articles/creating-an-issue/) for this project in the [GitHub Issues for the website](https://github.com/govdirectory/website/issues).
+
+You don't need to change any of our code or documentation to be a contributor!
+
+## Documentation and code in pull requests
+
+If you want to add to the documentation or code of one of our projects you should make a pull request.
+
+If you never used GitHub, get up to speed with [Understanding the GitHub flow](https://guides.github.com/introduction/flow/) or follow one of the great free interactive courses in the [GitHub learning lab](https://lab.github.com/) on working with GitHub and working with MarkDown, the syntax this project's documentation is in.
+
+This project is [licensed CC-0](LICENSE.md), which essentially means that the project, along with your contributions is in the public domain in whatever jurisdiction possible, and everyone can do whatever they want with it.
+
+### 1. Make your changes
+
+This project uses the [GitFlow branching model and workflow](https://nvie.com/posts/a-successful-git-branching-model/).
```

Comment:
Above we are referring to the GitHub flow here to GitFlow, they are different workflows. Rather than suggesting an entire model we should only suggest that one make changes to an working branch. Other aspects can be managed during actual pull requests.